protocol-dashboard/src/store/cache/analytics/hooks.ts

```diff
@@ -150,20 +155,40 @@ export function fetchApiCalls(
   }
 }
 
+/**
+ * Fetches time series data from a discovery node
+ * @param route The route to fetch from (plays, routes)
+ * @param bucket The bucket size
+ * @param clampDays Whether or not to remove partial current day
+ * @param node An optional node to make the request against
+ * @returns the metric itself or a MetricError
+ */
 async function fetchTimeSeries(
   route: string,
   bucket: Bucket,
-  clampDays: boolean = true
+  clampDays: boolean = true,
+  node?: string
 ) {
   const startTime = getStartTime(bucket, clampDays)
   let error = false
   let metric: TimeSeriesRecord[] = []
   try {
     const bucketSize = BUCKET_GRANULARITY_MAP[bucket]
-    const data = (await fetchWithLibs({
-      endpoint: `v1/metrics/${route}`,
-      queryParams: { bucket_size: bucketSize, start_time: startTime }
-    })) as any
+    let data
+    let endpoint = `${node}/v1/metrics/${route}?bucket_size=${bucketSize}&start_time=${startTime}` 
+    if (route === 'routes') {
```

Comment:
the original PR wanted to direct all `route`s to `${node}/v1/metrics/${route}?bucket_size=${bucketSize}&start_time=${startTime}`. however, `v1/metrics/routes` isn't supported anymore. the rest of the dashboard app seems to use `v1/metrics/aggregates/routes/<bucket>?bucketSize=...` so I added that here as well.

Here's an example response: https://discoveryprovider.staging.audius.co/v1/metrics/aggregates/routes/month?bucket_size=day

I'm not sure what the difference between `summed_unique_count` and `unique_count` is, but the overall unique users graph uses `summed_unique_count`. I used `unique_count` for the individual unique users graph and `total_count` for the individual API calls graph. is this correct?